apps/dashboard/lib/trpc/routers/ratelimit/query-overview-logs/index.ts

```diff
@@ -0,0 +1,133 @@
+import { ratelimitQueryOverviewLogsPayload } from "@/app/(app)/ratelimits/[namespaceId]/_overview/components/table/query-logs.schema";
+import { clickhouse } from "@/lib/clickhouse";
+import { db } from "@/lib/db";
+import { rateLimitedProcedure, ratelimit } from "@/lib/trpc/ratelimitProcedure";
+import { TRPCError } from "@trpc/server";
+import { type RatelimitOverviewLog, ratelimitOverviewLogs } from "@unkey/clickhouse/src/ratelimits";
+import { z } from "zod";
+import { transformFilters } from "./utils";
+
+const RatelimitOverviewLogsResponse = z.object({
+  ratelimitOverviewLogs: z.array(ratelimitOverviewLogs),
+  hasMore: z.boolean(),
+  nextCursor: z
+    .object({
+      time: z.number().int(),
+      requestId: z.string(),
+    })
+    .optional(),
+});
+
+type RatelimitOverviewLogsResponse = z.infer<typeof RatelimitOverviewLogsResponse>;
+
+export const queryRatelimitOverviewLogs = rateLimitedProcedure(ratelimit.update)
+  .input(ratelimitQueryOverviewLogsPayload)
+  .output(RatelimitOverviewLogsResponse)
+  .query(async ({ ctx, input }) => {
+    // Get workspace
+    const workspace = await db.query.workspaces
```

Comment:
would be nice to rewrite this to only load ratelimit namespace, instead of workspace + namespaces

ctx.workspace already contains a loaded workspace